benchmarks/imaptest/README.md

```diff
@@ -0,0 +1,41 @@
+# ImapTest  - Profiling & Compliance
+
+This "benchmark" uses [Dovecot's ImapTest tool](https://imapwiki.org/ImapTest) to profile the performance of Gluon. The 
+information present here can also be used to verify the compliance of Gluon with the IMAP protocol.
+
+## Installation
+
+Follow the instructions outlined in [the tools' installation page](https://imapwiki.org/ImapTest/Installation) 
+to build the binary. The test mailbox is already present in this folder.
+
+## Running ImapTest
+
+The bare minimum required for running ImapTest is a username and a password:
+
+```bash
+imaptest host=127.0.0.1 port=1143 user=user1@example.com pass=password1
+```
+
+For convenience, we have provided a few test scripts to quickly test Gluon with the demo binary present in this 
+repository.
+
+ * **default.sh**: Runs the default ImapTest benchmarks.
+ * **full.sh**: Runs every available ImapTest operation.
+
+Both of these scripts can be customized with the following environment variables:
+
+* **IMAPTEST_BIN**: Location of the ImapTest binary.
+* **SECS**: Number of seconds for which to run ImapTest.
```

Comment:
What's the minimum duration to set in order to have all test cases covered at least once ?

Reply:
It's hard to say. It's all based on the probability parameters you supply to the program.

If you run it for a couple of min you should be able to get good coverage. Failing that, you can also restrict the check list and run different sessions that test for different things.